packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -1065,13 +1061,22 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         view
         returns (RequestStatus)
     {
-        if (priceRequest.lastVotingRound == 0) return RequestStatus.NotRequested;
-        else if (priceRequest.lastVotingRound < currentRoundId) {
+        if (priceRequest.lastVotingRound == 0) {
+            return RequestStatus.NotRequested;
+        }
```

Comment:
small formatting nits that some of. these can be done on one line.

Reply:
Fixed: https://github.com/UMAprotocol/protocol/pull/4415/commits/097b280d10b086727b7ae3505badadf70219713c